website/docs/docs/contributing/adapter-development/2-building-a-new-adapter.md

```diff
@@ -363,6 +404,9 @@ While much of dbt's adapter-specific functionality can be modified in adapter ma
 
 </File>
 
+#### Grants Macros
+
+See [this GitHub discussion](https://github.com/dbt-labs/dbt-core/discussions/5468) for information on the macros required for `GRANT` statements: 
```

Comment:
- No colon at the end
- I think we do want information about the `grants` macros (and all other adapter-specific functionality) to eventually migrate here, into the docs, rather than GH discussions. (At the same time, we should look into ways to auto-generate much more of the adapter API spec.) What do you think?